adapter/persistence/persistence.go

```diff
@@ -0,0 +1,16 @@
+package persistence
+
+// DatastoreHandler
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported type DatastoreHandler should be of the form "DatastoreHandler ..." (with optional leading article)